dario_loan_api.raml

```diff
@@ -0,0 +1,78 @@
+#%RAML 1.0
+mediaType: application/json
+title: Loan API
+version: 1.0
+description: API to manage the loan payments control system from a fintech
+baseUri: http://localhost:3000/
+types:
+  loan_id:
+    pattern: ^([0-9]){3}-([0-9]){4}-([0-9]){4}-([0-9]){4}$   
+    example: 000-0000-0000-0000
+  payment:
+    type: object
+    properties:
+      status:
+        pattern: (made|missed)
+        example: made
+      date: date-only
+      amount: number
+  loan:
+    type: object
+    properties:
+      amount: number
+      term: number
+      rate: number
+      date: date-only
+/loans:
+  post:
+    description: Register a new loan
+    body:
+      type: [loan]
+    headers:     
+      amount: number
+      term: number
+      rate: number
+      date: date-only
+    responses: 
+      201:
+        description: Loan registered
+        headers:
+          Location:         
+            description: '/loans/{loan_id}/'
+      500:
+        description: Fail to register new loan. Please, try again later    
+  /{loan_id}/payments:
```

Comment:
I think loan id is more accurate.. its not the client id, for example, which could refer to the client registry in the CRM. Since the documentation is something to present to the stakeholder, could be interesting give this id the context of our API.